### DISCUSSION TITLE
Prettier Plugin TailwindCSS isn't working in NextJS

### DISCUSSION THREAD

**Discussion by zainsadaqat** [tailwindlabs/tailwindcss#12026] [Help] [answered]
I followed the instructions and ran the following command:

```
npm install -D prettier prettier-plugin-tailwindcss
```

But I didn't add the plugin to the Prettier config:
```js
// prettier.config.js
module.exports = {
  plugins: ['prettier-plugin-tailwindcss'],
}
```
The reason is I couldn't find the `prettier.config.js` file.

## Please guide me on how I can find prettier.config.js file and set the prettier plugin for tailwindcss up correctly.

**Comment by vishalvoid** [comment]
just install preetier as default foarmatter and without any extra changes it should work.

**Reply by vishalvoid** [reply]
in vs code. and set it as default formatter.

**Reply by zainsadaqat** [reply]
Hey @vishalvoid 👋 
Thanks for your response. Prettier is already set as default formatted but still it's not working. :(

**Comment by alex-krasikau** [comment, marked as answer]
I faced the same issue during my latest project setup.

@zainsadaqat Just create an empty prettier.config.js and set up the plugin as you mentioned in your message:
```
module.exports = {
  plugins: ['prettier-plugin-tailwindcss'],
}
```

After setting it up, it should start working.

Prettier should pick up the plugins from the `node_modules` without the need to configure them manually. However, I guess the issue here is that VSCode picks up the prettier package from another location (for example, from the global installation), which is why it doesn't recognize the plugin.

Let me know if it doesn't help, and we will delve deeper into this.

**Reply by zainsadaqat** [reply]
Wow!!! 🤩 
It's working now ✅ 

### Thank you so much @alex-krasikau You're a Genius Bro 👌 

<img src="https://media3.giphy.com/media/dIxkmtCuuBQuM9Ux1E/giphy.gif"/>
